eng/scripts/mirror-container-image.ps1

```diff
@@ -0,0 +1,23 @@
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$Image,
+    [Parameter(Mandatory=$true)]
+    [string]$Mirror,
+    [switch]$RegistryLogin
+)
+
+$ErrorActionPreference = 'Stop'
+$PSNativeCommandUseErrorActionPreference = $true
+
+if ($RegistryLogin) {
+    $mirrorRegistry = $Mirror.Split('.')[0]
+    Write-Host "Logging in to $mirrorRegistry"
+    az acr login -n $mirrorRegistry
+}
+
+Write-Host "docker pull $Image"
+docker pull $Image
+Write-Host "docker tag $Image $Mirror"
+docker tag $Image $Mirror
+Write-Host "docker push $Mirror"
+docker push $Mirror
```

Comment:
How likely would these commands fail? Should we check for exit codes so that we get a failing mirroring pipeline if they do fail?

Reply:
I set the `$PSNativeCommandUseErrorActionPreference` above so it will exit out, now that the variable ships with stock powershell.